lms/templates/fields/field_textarea.underscore

```diff
@@ -43,5 +43,13 @@
                 <span class="u-field-message-help" id="u-field-message-help-<%- id %>"> <%- message %></span>
             </span>
         <% } %>
+        <% if (mode === 'edit') { %>
+          <div class="profile-about-character-count">
```

Comment:
I'd name this ``field-textarea-character-count`` since this change could be used outside the learner profile.